### PR TITLE
FRENCH translation for QuickReply Reloaded extension

### PR DESCRIPTION
**Pull request by Galixte**

### CODE REVIEW

File: language/fr/quickreply.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+//end mod Translit
+//begin mod CapsLock Transfer
+	'QR_TRANSFORM_TEXT'			=> 'Modifier la casse du texte :',
+	'QR_TRANSFORM_TEXT_TOOLTIP'	=> 'Appuyer sur un bouton pour modifier la casse du texte sélectionné ou de tout le texte',
+	'QR_TRANSFORM_TEXT_LOWER'	=> '&#9660; abc',
+	'QR_TRANSFORM_TEXT_UPPER'	=> '&#9650; ABC',
+	'QR_TRANSFORM_TEXT_INVERS'	=> '&#9660;&#9650; aBC',
+	'QR_TRANSFORM_TEXT_LOWER_TOOLTIP'	=> 'minuscules',
+	'QR_TRANSFORM_TEXT_UPPER_TOOLTIP'	=> 'MAJUSCULES',
+	'QR_TRANSFORM_TEXT_INVERS_TOOLTIP'	=> 'inverser la casse',
```
Comment (lavigor) [inline language/fr/quickreply.php]: Original string `iNVERT cASE` means that case will be inverted.
Your string is fully lowercase (not the same as the original idea).

**Comment by lavigor** [comment]
Please note that current version is not fully ready: some new features will require additional language variables.
So it will be great if you can keep the translation up-to-date.

File: language/fr/quickreply.php (hunk 1 above)
Comment (Galixte) [inline language/fr/quickreply.php]: Ok i understand and i will change this for the next update, i’m not sure that this presentation has a impact on the reader.

**Comment by Galixte** [comment]
i always update my translation, be sure I will keep it.

**Pull request merged**